tests/go/cmd/kungfu-test-public-apis/kungfu-test-public-apis.go

```diff
@@ -5,33 +5,34 @@ import (
 	"flag"
 	"fmt"
 
-	kf "github.com/lsds/KungFu/srcs/go/kungfu"
-	kb "github.com/lsds/KungFu/srcs/go/kungfubase"
+	kb "github.com/lsds/KungFu/srcs/go/kungfu/base"
+	"github.com/lsds/KungFu/srcs/go/kungfu/peer"
+	"github.com/lsds/KungFu/srcs/go/kungfu/session"
 	"github.com/lsds/KungFu/srcs/go/utils"
 )
 
 func main() {
 	flag.Parse()
-	kungfu, err := kf.New()
+	p, err := peer.New()
 	if err != nil {
 		utils.ExitErr(err)
 	}
-	kungfu.Start()
-	defer kungfu.Close()
-	tests := []func(*kf.Kungfu){
+	p.Start()
+	defer p.Close()
+	tests := []func(*peer.Peer){
 		// TODO: more tests
 		testAllReduce,
 		testGetPeerLatencies,
 		testP2P,
 	}
 	for i, t := range tests {
 		fmt.Printf("# test: %d\n", i)
-		t(kungfu)
+		t(p)
 	}
 }
 
-func testGetPeerLatencies(kungfu *kf.Kungfu) {
-	sess := kungfu.CurrentSession()
+func testGetPeerLatencies(peer *peer.Peer) {
+	sess := peer.CurrentSession()
```

Comment:
What is the difference between peer.CurrentSession and defaultPeer.CurrentSession?

Reply:
`defaultPeer` is only used in C API.